content/guides/models/registry/search_registry.md

```diff
@@ -7,13 +7,89 @@ title: Find registry items
 weight: 7
 --- 
 
-Use the global search bar in the W&B Registry App to find a specific registry, collection, artifact version tag, collection tag, or alias. To search globally:
+Use the global search bar in the W&B Registry App or Python SDK to find a specific registry, collection, collection tag, artifact version name, artifact tag, or artifact alias.
 
-1. Navigate to the W&B Registry App at https://wandb.ai/registry/. 
+Only items that you have permission to view appear in the search results.
+
+## Interactively search for registry items
+
+To search globally:
+
+1. Navigate to the W&B Registry App.
 2. Specify the search term in the search bar at the top of the page. Press Enter to search.
 
 Search results appear below the search bar if the term you specify matches an existing registry, collection name, artifact version tag, collection tag, or alias.
 
-Only items that you have permission to view appear in the search results.
+{{< img src="/images/registry/search_registry.gif" alt="" >}}
+
+
+## Programmatically search for registry items
```

Comment:
Same feedback aBout the adverb here. Maybe move it to the end of the sentence in both cases.